xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/colorThemeInit.vm

```diff
@@ -72,6 +72,11 @@
 ##
 ## Overwrite with the values set in the custom theme
 #if (!$themeObj)
+## This algorithm for detecting the current color theme here is synchronized with the one in
+## org.xwiki.lesscss.internal.colortheme.CurrentColorThemeGetter.getCurrentColorTheme(boolean, String).
+## Don't customize it only in this file, it can cause LESS compilation cache issues.
+## See https://jira.xwiki.org/browse/XWIKI-21819 .
+##
```

Comment:
Thanks!
If this is just a doc update, no need to link the commit message to a specific issue, so I suggest renaming the commit message to something like "[Misc] Update doc for ..."
Otherwise, lgtm.

Reply:
I left the ticket in as the ticket has clear explanation and examples of how it can go wrong.
Is there a specific rule about not having such mentions in the comments?